go.mod

```diff
@@ -1,20 +1,37 @@
-module github.com/lockedinspace/letme
+module github.com/github.com/lockedinspace/letme
```

Comment:
Seems like the ``module github.com/github.com/lockedinspace/letme`` line has duplicated github.com entry.

Reply:
Fixed on next commit